### DISCUSSION TITLE
Syntax for defining multiple expected status codes for Http Request

### DISCUSSION THREAD

**Discussion by mremadevi** [elsa-workflows/elsa-core#5357] [Q&A]
What is the syntax for defining multiple expected status codes for Http Request ? Please share a sample

**Comment by glime-ai** [comment]
To define multiple expected status codes for an HTTP request in a workflow, you would typically use a collection or array to list all the acceptable status codes. However, the syntax for this can vary depending on the workflow system or programming language you are using.

In the context of Elsa workflows, based on the JSON snippets provided, it seems that status codes are set using the "StatusCode" property within a "WriteHttpResponse" activity. The "Literal" expression type is used to set a single status code, as seen in the following example:

```json
{
  "$id": "47",
  "name": "StatusCode",
  "expressions": {
    "$id": "48",
    "Literal": "OK"
  }
}
```

However, the provided context does not include a direct example of setting multiple status codes. If Elsa workflows support setting multiple expected status codes directly, it would likely involve using a different expression type that allows for arrays or collections. If such functionality is not directly supported, you might need to implement custom logic to handle multiple status codes.

Since the provided context does not contain a clear example of setting multiple status codes, and the official documentation or examples are not included in the context, I cannot provide a specific syntax or sample for Elsa workflows. You may need to refer to the official Elsa documentation or seek clarification from the Elsa community or support channels for the exact syntax.

If you are using a different system or programming language, please provide additional details so I can give you a more accurate answer.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
